<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOption</class>
 <widget class="QWidget" name="DisplayOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QToolButton" name="thinLinesButton">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Show invisible lines</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/app.qrc">
       <normaloff>:/app/icons/thinlines5.png</normaloff>:/app/icons/thinlines5.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="mirrorVButton">
     <property name="toolTip">
      <string>Vertical flip</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/app.qrc">
       <normaloff>:/app/icons/mirrorV.png</normaloff>:/app/icons/mirrorV.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="outLinesButton">
     <property name="toolTip">
      <string>Show outlines only</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/app.qrc">
       <normaloff>:/app/icons/outlines5.png</normaloff>:/app/icons/outlines5.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="mirrorButton">
     <property name="toolTip">
      <string>Horizontal flip</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/app.qrc">
       <normaloff>:/app/icons/mirror.png</normaloff>:/app/icons/mirror.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="onionPrevButton">
     <property name="toolTip">
      <string>Onion skin previous frame</string>
     </property>
     <property name="statusTip">
      <string>Onion skin previous frame</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/app.qrc">
       <normaloff>:/app/icons/onionPrev.png</normaloff>:/app/icons/onionPrev.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="onionNextButton">
     <property name="toolTip">
      <string>Onion skin next frame</string>
     </property>
     <property name="statusTip">
      <string>Onion skin next frame</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resource/app.qrc">
       <normaloff>:/app/icons/onionNext.png</normaloff>:/app/icons/onionNext.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/app.qrc"/>
 </resources>
 <connections/>
</ui>
